roles-in-tech/qa.md

```diff
@@ -2,89 +2,105 @@
-
-Make sure to mention these things:
-
-- Things
-	- This is a sub-thing
-- More things
-- Even more things
-- Even more things
-
-
-### Common Mistakes / Misconceptions
-
-List things that apprentices might not realize, might assume at first, or should avoid.
-
-- Example
-- Example
-
+- Go through [these slides](https://docs.google.com/presentation/d/17PG75lHEEOaoNM6OuAUeoaLa7Hf14tJPiwwdy5I-G-c/edit#slide=id.g22b045fc2c_0_8)  to get a genereal idea about a QA engineer and her job description. Follow with the lesson below.
+
+#### Who is a quality assurance engineer
+
+- Quality Assurance engineers aim to help create quality products. The main function of a QA engineer is to prevent defects, and therefore ensure the quality of the development process and its results.
+- Quality assurance engineer responsibilities are somewhere in-between of product manager's and developer's.
+- QAs put software to test while developers are busy writing more code. Their goal is to make every step of development trouble-free.
+
+#### Duties of a quality assurance engineer
+
+- **Establish and Maintain Quality Management Systems**
+    - QA engineers create the quality management policies and procedures that are followed by all employees who are a part of the production process.
+- **Develop and Monitor Performance Metrics for All Stages of the Production Process**
+    - QA engineers are responsible for figuring out how to quantify quality in all steps of the production process. This requires a deep understanding of their concerned domain.
+- **Determine Root Cause of Faulty Products**
+    - Deficient products aren’t just discarded and forgotten about. It is the responsibility of the QA engineer to figure out if there is a larger issue behind a dip in quality.
+- **Run Various Tests on Finished Products**
+    - QA engineers create and run the various tests necessary to prove the product performs as intended and meets the necessary standards. 
+
+#### What makes a good quality assurance engineer
+- Communicaton skills
+    - They need to effectively communicate with development team and not bully them with a deadline. 
+    - They also have to communicate with the customer to understand the concept of application.
+- Judgment skills are needed to assess high-risk areas of an application on which to focus testing efforts when time is limited.
+- A ‘test to break’ attitude, a strong desire for quality, and an attention to detail.
+
+#### How to become a QA engineer
+- Get all the skills
+    - Usually, employers ask and expect a bachelor’s degree from software quality engineers or quite a similar certification completed by the professionals. The certification or degree is a proof that you have enough fundamental skills to be a software quality engineer.
+    - You are strongly recommended to have any qualification and join the QA certification course to become a successful quality engineer.
```

Comment:
What certification course?

Reply:
What would you recommend to people who are reading this because they are taking a bootcamp or are self-taught-  how can they get into this career?